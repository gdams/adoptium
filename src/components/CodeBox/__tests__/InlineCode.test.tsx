import React from 'react';
import userEvent from '@testing-library/user-event';
import { render, screen } from '@testing-library/react';

import InlineCode from '../InlineCode';

describe('InlineCode component', (): void => {
  it('renders correctly', (): void => {
    const { container } = render(
      <InlineCode>
        This is some sample code
      </InlineCode>
    );
    expect(container).toMatchSnapshot();
  });
});