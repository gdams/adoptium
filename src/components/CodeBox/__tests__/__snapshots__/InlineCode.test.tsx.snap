// Vitest Snapshot v1

exports[`InlineCode component > renders correctly 1`] = `
<div>
  <code
    class="_code_4cf629"
  >
    This is some sample code
  </code>
</div>
`;
